prac_08/dynamic_labels.py

```diff
@@ -0,0 +1,31 @@
+"""
+Rhys Bush - Dynamic Labels
+Expected time: 5 minutes
+Actual time: 15 minutes
+"""
+
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.uix.label import Label
+
+
+class DynamicLabels(App):
+
```

Comment:
Missing class docstring and method docstrings for build(self) and create_labels(self)

Reply:
That is true - I tend to forget commenting.